### PR TITLE
publish result in a popup

### PR DESCRIPTION
**Pull request by josepablofm78**
## Description
It shows the result of the publish flow in a popup

### CODE REVIEW

File: client/src/+assets/user-publish-steps/prices.tsx [hunk 1]
```diff
@@ -79,23 +83,28 @@ export const PricesStep = (props: PricesProps) => {
...
                     <UiDivider/>
                     <UiFormGroup orientation={Orientation.Vertical}>
                             
                             {
-                                (!showForm) ?  <div className={b('user-publish-submit-container', ['updated-message'])}>
+                                (!showForm) ? 
+                              
+                                 <div className={b('user-publish-submit-container', ['updated-message'])}>
+                                  <ResultPopup  message={resultOk?assetMessage:errorAssetMessage} additionalMessage={filesUploadedMessage}  popupRef={resultPopupRef} resultOk= {resultOk}/>
+                                    {/*
```
Comment (dancespiele) [inline client/src/+assets/user-publish-steps/prices.tsx:107]: better to remove comment

File: client/src/+assets/user-publish-steps/main-page.tsx [hunk 2]
```diff
@@ -190,14 +196,18 @@ export const UserPublishMultiStep: NextPage = () => {
             onAssetPublish({nftAddress: getNftTierAddress(), metadata: generateMetadata()})
```
Comment (dancespiele) [inline client/src/+assets/user-publish-steps/main-page.tsx:196]: better to try to use async await, easier to read

File: client/src/+assets/user-publish-steps/prices.tsx (hunk 1 above)
Comment (josepablofm78) [reply client/src/+assets/user-publish-steps/prices.tsx]: done

File: client/src/+assets/user-publish-steps/main-page.tsx (hunk 2 above)
Comment (josepablofm78) [reply client/src/+assets/user-publish-steps/main-page.tsx:196]: in this specific case I prefer to use Promise syntax, is easier to handle the specific result of the call compare to await with try/catch

**Review by dancespiele — APPROVED** [review]

**Pull request merged**